scss/_patterns_accordion.scss

```diff
@@ -156,5 +156,9 @@
     &[aria-hidden='true'] {
       display: none;
     }
+
+    &.is-not-indented {
```

Comment:
@lyubomir-popov @sowasred2012 Any thoughts on naming this variant?

Reply:
`is-outdented`?

Reply:
@lyubomir-popov I guess the intention of this variant is to use it with tick elements (checkbox/radio) right? We don't want people to use it just because they don't want left padding?

In this case we could use this in the name `.has-tick-elements`?